src/components/BidList/BidList.tsx

```diff
@@ -90,6 +90,10 @@ function BidList(props: Props) {
             })
             setBids(bids);
 
+            if(bids.length < 10 && newBids.length !== 0){
```

Comment:
This change is redundant, please share code with AuctionList.tsx

Reply:
historisch bedingt